Frameworks/Foundation/NSInputStream.mm

```diff
@@ -19,174 +19,80 @@
 #import "Foundation/NSStream.h"
 #import "Foundation/NSString.h"
 #import "Foundation/NSInputStream.h"
-#import "NSStreamInternal.h"
 #import "LoggingNative.h"
 
-#ifdef WIN32
-#include <io.h>
-#elif defined(WINPHONE)
-#else
-//#include <unistd.h>
-#endif
+#import "NSCFInputStream.h"
 
 static const wchar_t* TAG = L"NSInputStream";
 
 @implementation NSInputStream
 
+BASE_CLASS_REQUIRED_IMPLS(NSInputStream, NSInputStreamPrototype, CFReadStreamGetTypeID);
+
 /**
  @Status Interoperable
 */
 + (id)inputStreamWithFileAtPath:(id)file {
```

Comment:
signatures etc etc. not sure whether you'll fix in the NSCF files but I think they should be correct in the NS files at least.